addons/project/views/project_project_views.xml

```diff
@@ -84,9 +84,10 @@
                         </page>
                         <page name="settings" string="Settings">
                             <group>
-                                <group name="analytic">
-                                    <div class="o_horizontal_separator mt-4 mb-3 text-uppercase fw-bolder small" groups="analytic.group_analytic_accounting">analytic</div>
-                                    <field name="account_id" class="oe_inline" groups="analytic.group_analytic_accounting"/>
+                                <field name="has_analytic_account" invisible="0"/>
+                                <group name="analytic" invisible="not has_analytic_account">
```

Comment:
Cannot we use `groups` attribute on `group` XML element?

Reply:
> Cannot we use `groups` attribute on `group` XML element?

here we can use groups. The issue will occur when the sale_project is installed.